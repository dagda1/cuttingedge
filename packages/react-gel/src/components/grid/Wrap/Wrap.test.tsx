import React from 'react';
import { shallow } from 'enzyme';
import { Wrap } from '.';

const wrap = (props = { children: <span>span</span> }) => shallow(<Wrap {...props} />);

describe('<Wrap />', () => {
  it('renders a div by default', () => {
    const wrapper = wrap();

    expect(wrapper.find('div')).toHaveLength(1);
  });

  it('renders a Wrap class', () => {
    const wrapper = wrap();

    expect(wrapper.find('.gel-wrap')).toHaveLength(1);
  });
});
