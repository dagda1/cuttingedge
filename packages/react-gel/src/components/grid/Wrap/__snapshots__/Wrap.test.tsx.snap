// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrap /> renders correctly 1`] = `
<div
  className="gel-wrap"
>
  <span>
    span
  </span>
</div>
`;
